azure-pipelines.yml

```diff
@@ -182,10 +205,7 @@ steps:
       curl -sSL https://raw.github.com/ndmitchell/hlint/master/misc/travis.sh | sh -s examples/mini-hlint/test/Main.hs
       curl -sSL https://raw.github.com/ndmitchell/hlint/master/misc/travis.sh | sh -s examples/mini-compile/src
   - bash: |
-      stack exec -- pacman -Syu --noconfirm
+      stack exec --stack-yaml $(stack-yaml) --resolver $(resolver) -- pacman -Syu --noconfirm
     condition: eq(variables['Agent.OS'], 'Windows_NT')
   - bash: |
-      stack setup --stack-yaml $(stack-yaml) --resolver $(resolver) > /dev/null
-      stack install --stack-yaml $(stack-yaml) --resolver $(resolver) alex happy > /dev/null
```

Comment:
this is done in CI.hs and has no (good) effect here

Reply:
hadrian has a configure step which requires some version of alex, happy be installed before it runs. we satisfy this in CI.hs with a `stack build alex happy`. i think this line of code we are removing was a failed experiment that never got cleaned up